homeassistant/components/gpsd/sensor.py

```diff
@@ -103,7 +109,7 @@ def native_value(self):
         return None
 
     @property
-    def extra_state_attributes(self):
+    def extra_state_attributes(self) -> dict[str, str]:
```

Comment:
When I look at the returned values, are latitude, longitude etc all returned as strings? If they are not strings, please change to str, Any